crates/stellar-scaffold-cli/README.md

```diff
@@ -82,15 +82,18 @@ The build process ensures:
 
 ### Setting contract metadata
 
-Contract metadata is set when running `stellar scaffold build` and adds the following fields from your contract's Cargo.toml to the contract metadata.
-
-| Metadata   | Cargo.toml Key (under `package`) | Description                                      |
-|------------|----------------|--------------------------------------------------|
-| `name`     | `name`         | The name of the contract                         |
-| `binver`   | `version`      | The version of the WASM bytecode of the contract                      |
-| `authors`  | `authors`      | The author(s) of the contract, often with an email  |
-| `home_domain` | `homepage`  | The relevant domain to relate to this contract   |
-| `source_repo` | `repository` | The source repository URL for the contract      |
+Contract metadata is set when running `stellar scaffold build`. You can configure metadata with 
+`[package.metadata.stellar]` section in your `Cargo.toml` file.
+For example:
+```toml
+[package.metadata.stellar]
+# When set to `true` will copy over [package] section's `name`, `authors`, `homepage`, `repository` and `version` (renamed to `binver` to comply with SEP-47)
+cargo_inherit = true
+# Override one of the inherited values
+name = "my-awesome-contract"
+homepage = "ahalabs.dev"
```

Comment:
Hmm, I like the inherit, but I thought moving things to our own section means we'd use our own key names, rather than mapping them. meaning we would use source_repo, home_domain, binver… That the names are different is of course an upstream unpleasantness, what do you think about this approach?

Reply:
I did the renaming in the implementation because of using [package] keys … I would think if we're moving to our own metadata section, we don't need to rename